server/src/internalClusterTest/java/org/opensearch/action/admin/ClientTimeoutIT.java

```diff
@@ -152,7 +157,6 @@ public void testRecoveriesWithTimeout() {
         assertThat(recoveryResponse.getShardFailures()[0].reason(), containsString("ReceiveTimeoutTransportException"));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/6255")
```

Comment:
From https://github.com/opensearch-project/OpenSearch/issues/6259, the underlying cause of `testSegmentReplicationStatsWithTimeout` test failure is `org.apache.lucene.store.LockObtainFailedException` exception, on underlying store during round of segrep. Can you verify this test in loop and ensure it is not flaky ?

Reply:
The test from #6259  `testSegmentReplicationStatsWithTimeout` causing `org.apache.lucene.store.LockObtainFailedException` is fixed with addition of :
```
    @Override
    protected boolean addMockInternalEngine() {
        return false;
    }
```
Yes from changes made in this PR this issue is fixed. I ran this test in loop for 500 iterations and seen no failure happening